### DISCUSSION TITLE
Unalbe to use Nifty Prediction using Artifical Intelligence and Live Index Scan : 5 EMA for Intraday

### DISCUSSION THREAD

**Discussion by anirbanphys** [pranjal-joshi/Screeni-py#148] [Q&A] [answered]
Dear Pranjal, 

I would like to thank you very much for sharing this wonderful, very well constructed tool with everyone. Particularly since it comes from India. 
This tool has just been downloaded and I was playing around with it to get an idea of its capabilities. 
I have attached the snippets for the first two features that aren't working for me. Can you please point me what is wrong here?

![E-1](https://user-images.githubusercontent.com/121126251/208919723-1a09fe3d-935e-42b8-b8a3-abe4aeea2184.png)

![E-2](https://user-images.githubusercontent.com/121126251/208919928-a8ee0e9b-acb2-43d9-aa61-1f576c27708a.png)

and

![E-3](https://user-images.githubusercontent.com/121126251/208920157-49cc5ab4-4417-4cd8-8e9b-d5ec944581f4.png)

Can you please guide me what is wrong here?

Thanks,
Anirban

**Comment by pranjal-joshi** [comment, marked as answer]
Hi @anirbanphys 

It seems like the screenipy is not able to download the required ML model files from the GitHub repo!

Please check your internet connectivity or if you have enabled any proxies on your system due to the access control is setup by your organization then disable it or try on your personal PC.

If that doesn't work then download following files and keep it in the same folder where `screenipy.exe` is kept.

https://github.com/pranjal-joshi/Screeni-py/blob/main/src/ml/nifty_model.h5

https://github.com/pranjal-joshi/Screeni-py/blob/main/src/ml/nifty_model.pkl

**Reply by anirbanphys** [reply]
Thank you @pranjal-joshi. Not sure why this was not working. I am running the utility in Ubuntu. I am running on my personal machine, but still this was not working. While I switched to moble network from broad band, NIFTY prediction worked fine. But still facing issues with "Live Intraday Scanner".

Thanks,
Anirban

**Reply by pranjal-joshi** [reply]
Hi @anirbanphys 

The Live Intraday scanner table will stay empty unless it detects 5-EMA entries during 9.15 to 3.30
This won't print table of stocks as it's continuously scanning Nifty & BankNifty only.
So you can try running this during market hours. You may get max 1-2 entries per day!

**Reply by anirbanphys** [reply]
Thank you for your reply.

**Comment by JobForBob** [comment]
Hey, there seems to be an issue in the AI gap prediction where nifty gapdown chances are locked to 17.707% and cannot go above. The gap up chances are fine but for the last few weeks, the gap down % is capped out at 17.707%
